docroot/themes/humsci/humsci_basic/src/scss/components/_masthead.scss

```diff
@@ -1,3 +1,46 @@
-.su-masthead .su-lockup {
-  z-index: $hb-z-index-lockup;
+.su-masthead {
+  .su-lockup {
+    z-index: $hb-z-index-lockup;
+  }
+
+  .su-site-search {
+    display: block;
+    padding-top: hb-calculate-rems(12px);
+    padding-bottom: hb-calculate-rems(28px);
+    max-width: hb-calculate-rems(265px);
```

Comment:
let's put this in settigs/variables.general.scss and make a named variable for 256px

Reply:
I placed it in `_variables/forms` instead. Let me know if that's ok.